test/TestApps/PushNotificationsTestApp/main.cpp

```diff
@@ -114,17 +114,26 @@ bool MultipleChannelRequestUsingMultipleRemoteId()
 
 bool ActivatorTest()
 {
-    PushNotificationManager::UnregisterActivator(PushNotificationRegistrationActivators::PushTrigger | PushNotificationRegistrationActivators::ComActivator);
-    g_registeredActivator = false;
-
+    PushNotificationManager::UnregisterAllActivators();
     try
     {
-        PushNotificationActivationInfo info(
-            PushNotificationRegistrationActivators::PushTrigger | PushNotificationRegistrationActivators::ComActivator,
-            c_fakeComServerId);
+        if (g_isPackaged)
+        {
```

Comment:
I think I would rather have two tests here, maybe: ComActivatorTest and ProtocolActivatorTest.

Reply:
It seems like the structure of our TAEF testing is to call the same unit test under pkg/unpkg scenarios. Should we just split up all tests into pkg/unpkg?

Reply:
Switching to IsActivatorSupported and leaving current impl.